src/concepts/essentials/httpql.md

```diff
@@ -2,99 +2,167 @@
-- **ncont**: Doesn't contain `value` (case insensitive).
-- **ne**: No equal to `value`.
-- **nlike**: SQLITE `NOT LIKE` Operator, see `like` for more details.
+::: tip TIPS
+
+- The `cont` and `ncont` Operators are case insensitive.
+- In SQLite - the `%` character matches zero or more characters (_such as `%.js` to match `.map.js`_) and the `_` character matches one character (_such as `v_lue` to match `vAlue`_).
+- The `like` Operator is case sensitive for unicode characters that are beyond the ASCII range.
+:::
 
 ### Regex
 
-This category of Operators works on Fields that are text values (or bytes) like `path` and `raw`.
+This category of Operators works on Fields that are text or byte values (_such as `path` and `raw`_).
+
+- `regex`: Matches the regex `/value.+/`.
+- `nregex`: Doesn't match the regex `/value.+/`.
+
+::: info
+Not all regex features are currently supported by Caido (_such as look-ahead expressions_) as they are not included in the regex library of Rust.
+:::
 
-- **regex**: Matches the regex `/value.+/`.
-- **nregex**: Doesn't match the regex `/value.+/`.
+::: tip
+Visit [https://regex101.com/](https://regex101.com/) and select **Rust** syntax for assistance in creating expressions.
+:::
 
-## 4. Value
+## Value
 
-This is the value against which the Field will be compared. The value is either an integer (like `1`), a string (`"value"`) or a regex (`/value/`) depending on the Field and Operator.
+This is the value against which the Field will be compared. The value type is either an **integer**, a **string** or a **regex** depending on the Field and Operator.
 
-### Preset
+### preset
 
-The `Preset` value is a different. You can reference Presets in one of two ways:
+When using the `preset` Namespace - the value can be reference by either the `Name` or the `Alias` of the Preset:
 
 - **Name**: `preset:"Preset name"`.
 - **Alias**: `preset:preset-alias`.
 
 View the [Filters](/reference/features/overview/filters.md) documentation for more information.
 
+### source
+
+- `intercept` - Traffic that has been proxied by Caido.
+- `replay` - Traffic generated by sending requests using [Replay](/reference/features/testing/replay.md).
+- `automate` - Traffic generated by an [Automate](/reference/features/testing/automate.md) campaign.
+- `workflow` - Traffic generated by a [Workflow](/reference/workflows/workflows.md).
+
+::: info
+Autocomplete is not currently available when using the `source` Namespace.
+:::
+
+::: tip
+
+- When using the `source` Namespace - use all lowercase characters when naming the desired Caido feature.
+- If you are not receiving results of a `source` query - click the `Advanced` settings button next to the HTTPQL query bar to ensure the desired `Sources` are enabled.
```

Comment:
Thats a good point, I would say the advanced is kinda of the same thing as the httpql syntax. We added the syntax for places where you dont have this advanced button like filter preset. So we should precise that tip is only valid for HTTP History and Search.

Reply:
`source` is not available as a query in HTTP History unless I am mistaken.